fiftyone/utils/voc.py

```diff
@@ -525,6 +525,13 @@ def from_annotation_dict(cls, d):
         """
         name = d["name"]
         bndbox = VOCBoundingBox.from_bndbox_dict(d["bndbox"])
+
+        # Handles CVAT exported attributes
+        if "attributes" in d:
```

Comment:
I threw in these `if "attributes" in d` checks just to avoid a bunch of operations on empty dictionaries that would happen in the 99.999% of use cases where CVAT-style attributes are not involved